cli.mjs

```diff
@@ -185,22 +242,23 @@ async function deploymentPrompt() {
 }
 
 /** Prompt the user to enter the ID of a study */
+// helper to check if the given study (input) is in firestore
+async function validateStudyFirebase(input) {
+  // subcollection is programmatically generated, if it doesn't exist then input must not be a valid studyID
+  const studyIDCollections = await getStudyRef(input).listCollections();
+  return studyIDCollections.find((c) => c.id === PARTICIPANTS_COL);
+}
+
 async function studyIDPrompt() {
   const invalidMessage = "Please enter a valid study from your Firestore database";
-  const validateStudyFirebase = async (input) => {
-    // subcollection is programmatically generated, if it doesn't exist then input must not be a valid studyID
-    const studyIDCollections = await getStudyRef(input).listCollections();
-    return studyIDCollections.find((c) => c.id === PARTICIPANTS_COL) ? true : invalidMessage;
-  };
-
   return await input({
     message: "Select a study:",
     validate: async (input) => {
       if (!input) return invalidMessage;
-
       switch (DEPLOYMENT) {
         case "firebase":
-          return validateStudyFirebase(input);
+          const res = await validateStudyFirebase(input);
+          return !res ? invalidMessage : true;
```

Comment:
We're calling `.find` inside `validateStudyFirebase` so I would rename this to be `studyCollection` or something like that? Because the return of the function is really the Firebase study collection or `undefined`